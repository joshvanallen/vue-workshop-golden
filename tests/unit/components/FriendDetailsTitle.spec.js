import { shallowMount } from "@vue/test-utils";
import FriendDetailsTitle from "@/components/FriendDetailsTitle.vue";

describe("FriendDetailsTitle.vue", () => {
  it("renders full name when passed prop.friend", () => {
    const friend = {
      firstName: 'John',
      lastName: 'Doe'
    };
    const wrapper = shallowMount(FriendDetailsTitle, {
      propsData: { friend }
    });
    expect(wrapper.text()).toMatchSnapshot();
  });
});
