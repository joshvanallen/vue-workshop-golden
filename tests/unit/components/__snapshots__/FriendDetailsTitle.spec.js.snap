// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendDetailsTitle.vue renders full name when passed prop.friend 1`] = `"John Doe"`;
